### PR TITLE
Service 11

### PR DESCRIPTION
**Pull request by michalkocon**

### CODE REVIEW

File: asn1-pusc-lib.pro [hunk 1]
```diff
@@ -271,6 +269,73 @@ DISTFILES += \
...
+    service-11/Subschedule.acn \
+    service-11/Subschedule.asn1 \
+    service-11/Group.acn \
+    service-11/Group.asn1 \
+    common/Source.acn \
+    common/Source.asn1 \
+    service-11/ST-11-Request.acn \
+    service-11/ST-11-Request.asn1 \
+    service-01/ST-01-Request.acn \
+    service-01/ST-01-Request.asn1 \
```
Comment (hcorg) [inline asn1-pusc-lib.pro]: new convention... isn't there any other way to distinguish those requests?

Comment (hcorg) [reply asn1-pusc-lib.pro]: (or we need to follow that convention with all files :( )

File: service-11/PUS-11-1.acn [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-11-1 DEFINITIONS ::= BEGIN
+
+TC-11-1-EnableTheTimebasedScheduleExecutionFunction []
```
Comment (hcorg) [inline service-11/PUS-11-1.acn]: I always wonder - should we keep words like "the" in names (to be more word-to-word with standard) or should we skip them to make our ids little shorter

File: service-11/PUS-11-13.asn1 [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-11-13 DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    Subschedule-ID FROM Subschedule
```
Comment (hcorg) [inline service-11/PUS-11-13.asn1]: SubSchedule or Subschedule? just wondering :D

File: service-11/PUS-11-13.asn1 [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+maxTM-11-13-ReportCount INTEGER ::= 4095
+
+TM-11-13-TimebasedScheduleSummaryReport ::= SEQUENCE
+{
+--! n UnsignedInteger, moved to acn
+    report SEQUENCE (SIZE(0 .. maxTM-11-13-ReportCount)) OF TM-11-13-Report
+}
+
+TM-11-13-Report ::= TM-11-13-ReportGeneric {Subschedule-ID, Group-ID, AbsoluteTime, ST-11-Request-ID}
```
Comment (hcorg) [inline service-11/PUS-11-13.asn1]: `OPTIONAL`s should be enabled or disabled in default implementation (or encoding will be incorrect - additional bits will be added to mark fields as present or not)

use `WITH COMPONENTS...` construct

File: service-11/PUS-11-13.asn1 (hunk 3 above)
Comment (michalkocon) [reply service-11/PUS-11-13.asn1]: Had the same problem :D. SubSchedule just looks weird to me, same as TimeBased

Comment (hcorg) [reply service-11/PUS-11-13.asn1]: how it's written in docs? if as two separate words, then SubSchedule, single word - single camel ;)

Comment (michalkocon) [reply service-11/PUS-11-13.asn1]: Sub-schedule and time-based, but making an "-" looks very inconsistent when using it in the middle of message's name

File: service-11/PUS-11-1.acn (hunk 2 above)
Comment (michalkocon) [reply service-11/PUS-11-1.acn]: We might as well leave it as just TC-11-1 and move full name to the comment

Comment (hcorg) [reply service-11/PUS-11-1.acn]: nah, I just wondered :) leave it as is

File: service-11/PUS-11-13.asn1 (hunk 3 above)
Comment (hcorg) [reply service-11/PUS-11-13.asn1]: so we should go with SubSchedule - it's two words :/.

**Review by hcorg — APPROVED** [review]

**Pull request merged**